core/components/molecules/sidesheet/__stories__/CustomHeader.story.jsx

```diff
@@ -50,10 +50,10 @@ export const customHeader = () => {
         dimension="regular"
         header={
           <div className="pl-7">
-            <Heading size="s">Untitled document</Heading>
-            <div className="d-flex">
+            <Heading size="m">Untitled document</Heading>
+            <div className="d-flex mt-2">
               <div className="mr-3">
-                <Badge appearance="primary">User Interface</Badge>
+                <Badge>User Interface</Badge>
```

Comment:
Please provide classname `mr-3` directly to the Badge component instead of wrapping it inside multiple `div`

Reply:
Done